src/overlays/actors/ovl_En_Prz/z_en_prz.c

```diff
@@ -71,45 +93,442 @@ const ActorInit En_Prz_InitVars = {
     (ActorFunc)EnPrz_Draw,
 };
 
-#endif
+AnimationHeader* D_80A77240[] = { &object_pr_Anim_004340, &object_pr_Anim_004274 };
+
+u8 D_80A77248[] = { 0, 0 };
+
+Vec3f D_80A7724C = { 0.0f, 0.0f, 0.0f };
+
+void EnPrz_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnPrz* this = THIS;
+
+    this->unk_20C = 0.01f;
+    this->unk_1E4 = this->actor.world.rot.y;
+
+    Actor_SetScale(&this->actor, 0.01f);
+
+    this->actor.targetMode = 3;
+    this->unk_1E8 = 0xFF;
```

Comment:
```suggestion
    this->unk_1E8 = 255;
```
Used as a color so decimal can be used